modules/installation-special-config-storage.adoc

```diff
@@ -158,7 +158,7 @@ When the `Do you wish to trust these keys? [ynYN]` prompt displays, type `Y`.
 +
```

Comment:
Noting Benjamin's comment here; I think it could be written as something like:

`.. If the nodes are configured with static IP addressing, use the `coreos-installer --append-karg` option, or run `coreos-installer iso customize --dest-karg-append`, when installing {op-system} nodes to set the IP address of the installed system.`